tests/import_specifications/test_data/manifest_small.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
This doesn't seem to match the documentation in individual_parsers but I assume the format is still changing